docs/src/content/docs/reference/helm-chart-config.mdx

```diff
@@ -434,6 +434,51 @@ For production environments, these should always be set to false. Instead, exter
             <td></td>
             <td>An object where the keys are the organism IDs and values are an [Organism (type)](#organism-type)</td>
         </tr>
+        <tr>
+            <td>`lineageSystemDefinitions`</td>
+            <td>Object</td>
+            <td></td>
+            <td>An object where the keys are the lineage system names and values are links to lineage system definition files per pipeline version (See [Lineage system defintions](#lineage-type))</td>
+
+        </tr>
+    </tbody>
+</table>
+
+### Lineage system definitions
+
+Here's an example of a `lineageDefinitions` section:
+
+```yaml
+lineageSystemDefinitions:
+  pangoLineage:  # Lineage name to use in metadata fields
+    "1": https://example.org/lineage_defintions_v1.yaml  # Definition per pipeline version
```

Comment:
Does this need to be quoted btw? I presume so?

Reply:
good point, they actually don't need to be. I was just overly cautious 😅 

I'll remove it.

Reply:
[`25ca1cf` (#3434)](https://github.com/loculus-project/loculus/pull/3434/commits/25ca1cfd34f650a7c315064d447caaaf6d81c063)